include/drivers/led.h

```diff
@@ -61,10 +87,13 @@ typedef int (*led_api_off)(struct device *dev, uint32_t led);
  * This is the mandatory API any LED driver needs to expose.
  */
 __subsystem struct led_driver_api {
+	/* Mandatory callbacks. */
 	led_api_blink blink;
 	led_api_set_brightness set_brightness;
 	led_api_on on;
 	led_api_off off;
+	/* Optional callbacks. */
+	led_api_get_info get_info;
```

Comment:
I don't see this mentioned in the RFC description or the commit log, but I assume you will update drivers/led/led_handlers.c for the new syscalls before moving this out of RFC, right? These do the necessary security checks when user mode is turned on.

Reply:
Yes I will.

Reply:
>  I assume you will update drivers/led/led_handlers.c for the new syscalls before moving this out of RFC, right?

The implementations look good to me.